models/lorenz_96/README.rst

```diff
@@ -0,0 +1,135 @@
+################
+Lorenz 96 README
+################
+
+Contents
+========
+
+#. `Overview`_
+#. `Namelist`_
+#. `Terms of Use`_
+#. `References`_
```

Comment:
Again ... workshop_setup.csh and used extensively in the DART_tutorial - and DART_LAB.

Reply:
Since the stated purpose of this pull request is to convert the documentation from HTML to reStructuredText, I think it makes sense to add the workshop_setup.csh descriptions in a different pull request.